playground/mocks/data/idp/idx/authenticator-verification-phone-options-voice-only.json

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
`authenticator-verification-phone-options-*`
`options` in the name kind of implying it's for authenticator list page.
would suggest to ditch it or maybe rename to `authenticator-verification-data-phone-voice-only`.
same to other new mock file that have `options` in name.